docs/howto/account-billing/manage-invoices.md

```diff
@@ -31,3 +31,13 @@ There is also the _Pay this invoice_ option.
 Choose it for an outstanding invoice, and you will get a pop-up with a detailed rundown of the dues, together with available payment methods.
 
 ![Payment options](assets/mgmnt-inv/shot-04.png)
+
+## Activate automatic payments
+
+You may activate automatic payments for pending invoices.
+From the left-hand side vertical pane, click on *Settings* and then on *Manage Invoice Settings*.
```

Comment:
I'd add a newline between the first and second sentence here.
```suggestion
You may activate automatic payments for pending invoices.

From the left-hand side vertical pane, click on *Settings* and then on *Manage Invoice Settings*.
```
What do you think?

Reply:
It's better, yes.